python/python/bystro/proteomics/fragpipe_tandem_mass_tag.py

```diff
@@ -0,0 +1,53 @@
+"""Load and prep fragpipe tandem mass Tag datasets."""
+
+from dataclasses import dataclass
+
+import pandas as pd
+
+ABUNDANCE_COLS = ["Index", "NumberPSM", "ProteinID", "MaxPepProb", "ReferenceIntensity"]
+ANNOTATION_COLS = ["plex", "channel", "sample", "sample_name", "condition", "replicate"]
+
+
+@dataclass(frozen=True)
+class TandemMassTagDataset:
+    """Represent a Fragpipe Tandem Mass Tag dataset."""
```

Comment:
It seems like there are several TMT outputs from FragPipe - do they all have the same type of output files and headings or should we be more specific here (i.e. TMT-10, TMT16)?

Reply:
The TMT dataset we have access to is described as 4-plex, but in the paper, it seems, it's described as TMT-10 (https://www.sciencedirect.com/science/article/pii/S0092867419311237?via%3Dihub).  I'm not sure how to resolve that apparent tension, but in any case this is the only fragpipe TMT dataset we have to work with at present.  

I can doublecheck this.